Firebase/Core/FIROptions.m

```diff
@@ -108,6 +109,21 @@ + (FIROptions *)defaultOptions {
 
 #pragma mark - Private class methods
 
++ (void)load {
+  // Report FirebaseCore version for useragent string
+  NSRange major = NSMakeRange(0, 1);
+  NSRange minor = NSMakeRange(1, 2);
+  NSRange patch = NSMakeRange(3, 2);
+  [FIRApp
+      registerLibrary:@"FirebaseCore"
```

Comment:
Let's change this to "fire-ios" to make it compact and generic (other examples could be fire-unity and fire-cpp)

Reply:
Done.